.github/workflows/changelog.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
         with:
           fetch-depth: 0
       - name: Setup Go
-        uses: actions/setup-go@v3
+        uses: actions/setup-go@v4
```

Comment:
v4 enables cache https://github.com/actions/setup-go#v4.

I wonder if we should remove the `go-cache`  step below. The v4 should use the `actions/cache`